src/Stratis.Bitcoin/Utilities/JsonConverters/Serializer.cs

```diff
@@ -21,19 +25,12 @@ public static void RegisterFrontConverters(JsonSerializerSettings settings, Netw
             settings.Converters.Add(new DateTimeToUnixTimeConverter());
             settings.Converters.Add(new TxDestinationJsonConverter());
             settings.Converters.Add(new LockTimeJsonConverter());
-            settings.Converters.Add(new BitcoinStringJsonConverter()
-            {
-                Network = network
-            });
-            settings.ContractResolver = new CamelCasePropertyNamesContractResolver();
-        }
+            settings.Converters.Add(new BitcoinStringJsonConverter(network));
 
-        public static T ToObject<T>(string data)
-        {
-            return ToObject<T>(data, null);
+            settings.ContractResolver = new CamelCasePropertyNamesContractResolver();
         }
 
-        public static T ToObject<T>(string data, Network network)
+        public static T ToObject<T>(string data, Network network = null)
```

Comment:
Ah mate can we remove these null defaults? I really want us to move away from these 👍

Reply:
We don't assign a default Network if it's null and having a null Network is valid as well.
Do you still want me to make it non-null?

Reply:
Hmm if there's no need to make it null then I would say leave it as non-null? If the need arises we can have overloads.

Reply:
I think it's ok to leave it like this.
In Stratis.Bitcoin.Features.Api.Startup we call the Serializer.RegisterFrontConverters method but we don't have a Network.